### PR TITLE
LPS-167073 HR_15 When user activates "Page Audit" button and Side panel pop-up, Focus is not moving to Side panel window

### PR DESCRIPTION
**Pull request by veroglez**
Hi guys! I send you the fix for: https://liferay.atlassian.net/browse/LPS-167073

The problem is mainly that, when the operating system's reduce motion is activated, the focus was trying to redirect to the panel when the panel still had a `visibility: hidden` style. 

**What has been fixed?**
- First, the classes necessary to open the panel are set and then the focus is redirected
- Focus is only redirected to the trigger if the active element is a close button.

**What needs to be reviewed?**
- The following commits: 236ac6dce2dc9cbd09ea2a6834da4514d1036f4c and 8c19999e989d9dca5e878b5ccf85afb69cd5599e

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js [hunk 1]
```diff
@@ -351,8 +351,11 @@ SideNavigation.prototype = {
 
 	_focusTrigger() {
 		const toggler = this.toggler;
+		const activeElementIsCloseButton = document.activeElement.classList.contains(
+			'sidenav-close'
+		);
```
[the remark this answers is not in the document]
Comment (veroglez) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js]: done!

**Review by markocikos — APPROVED** [review]
@veroglez LGTM! :sparkles:

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
@markocikos, could someone from QA see if the failures are related? Thanks in advance!

**Comment by JerryAdrianoo** [comment]
Hey, @veroglez  I'm going to run ci:relevant again because in your PR, only one test failed, but the probability of it being related is very high! But I downloaded the bundle and tested it locally, but the test failed again. I'll run ci:relevant again to be more sure.

**Comment by JerryAdrianoo** [comment]
ci:test:relevant

**Comment by JerryAdrianoo** [comment]
I found some issues related to your changes! Look below.

```
console.error ../../../node_modules/react/node_modules/prop-types/checkPropTypes.js:20
     [exec]       Warning: Failed prop type: The prop `name` is marked as required in `BaseEditor`, but its value is `undefined`.
     [exec]           in BaseEditor (created by ForwardRef)
     [exec]           in ForwardRef (created by RichText)
     [exec]           in div (created by ClayInputGroupItem)
     [exec]           in ClayInputGroupItem (created by RichText)
     [exec]           in div (created by ClayInputGroup)
     [exec]           in ClayInputGroup (created by RichText)
     [exec]           in div (created by ClayFormGroup)
     [exec]           in ClayFormGroup (created by FieldBase)
     [exec]           in FieldBase (created by RichText)
     [exec]           in RichText (created by FieldLazy)
```

And others

     [exec]     console.error ../../../node_modules/react-dom/node_modules/prop-types/checkPropTypes.js:20
     [exec]       Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
     [exec]           in input (created by ClayInput)
     [exec]           in ClayInput (created by DocumentLibrary)
     [exec]           in div (created by ClayInputGroupItem)
     [exec]           in ClayInputGroupItem (created by DocumentLibrary)
     [exec]           in div (created by ClayInputGroup)
     [exec]           in ClayInputGroup (created by DocumentLibrary)
     [exec]           in div (created by DocumentLibrary)
     [exec]           in DocumentLibrary (created by DocumentLibrary)
     [exec]           in div (created by ClayFormGroup)
     [exec]           in ClayFormGroup (created by FieldBase)
     [exec]           in FieldBase (created by DocumentLibrary)
     [exec]           in DocumentLibrary (created by DocumentLibraryWithProvider)
     [exec]           in PageProvider (created by DocumentLibraryWithProvider)
     [exec]           in DocumentLibraryWithProvider
     [exec] 
     [exec]   ??? Console
     [exec] 
     [exec]     console.error ../../../node_modules/react/cjs/react.development.js:167
     [exec]       Warning: Each child in a list should have a unique "key" prop.
     [exec]       
     [exec]       Check the render method of `ForwardRef`. See https://fb.me/react-warning-keys for more information.
     [exec]           in Fragment (created by ForwardRef)
     [exec]           in ForwardRef (created by ForwardRef)
     [exec]           in ForwardRef (created by Select)
     [exec]           in div (created by Select)
     [exec]           in TooltipProvider (created by Select)
     [exec]           in Select (created by Select)
     [exec]           in div (created by ClayFormGroup)
     [exec]           in ClayFormGroup (created by FieldBase)
     [exec]           in FieldBase (created by Select)
     [exec]           in Select (created by SelectWithProvider)
     [exec]           in PageProvider (created by SelectWithProvider)
     [exec]           in SelectWithProvider
     [exec] 
     [exec]   ??? Console
     [exec] 
     [exec]     console.error ../../../node_modules/react-dom/cjs/react-dom.development.js:530
     [exec]       Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
     [exec]           in input (created by ClayInput)
     [exec]           in ClayInput (created by ValidationDate)

Can you take a look, please? These errors are in js-unit-jdk8/0/0

**Comment by veroglez** [comment]
Hi @JerryAdrianoo, what you mention doesn't seem to be errors, they are just warnings, the tests should not fail because of this. Additionally, I have checked locally all the frontend tests of the modified modules and it seems that they all pass.

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**